config/job_conf.xml

```diff
@@ -59,6 +59,14 @@
       <param id="docker_tag_override">v1.2.12.0_cv1.0.65</param>
       <param id="max_pod_retrials">3</param>
       <param id="docker_enabled">true</param>
+    </destination>
```

Comment:
Thanks @vagelishand! This setup is no longer the preferred one, you should be using what is described here: https://github.com/phnmnl/phenomenal-h2020/wiki/Setting-up-Galaxy-wrappers-on-PhenoMeNal-Galaxy-Container#tool-cpu-and-memory-usage-requests-and-limits

Reply:
Hello Pablo. So the docker_tag_override that is "v1.2.12.0_cv1.0.65" has to be change to "latest"?

Reply:
No, the tag in this case is fine, is how the destination is handled. See the link I provided on my previous comment, let me know if you have any doubts.

Reply:
Hi @vagelishand using the release tag was the correct way to go, please don't change it to the `dev_` tag.

Reply:
or do you need a different version of Batman for this? Why are you changing Batman's tag if this is a pull request for bruker2batman?